bioio_tiff_glob/reader_metadata.py

```diff
@@ -12,16 +12,16 @@ class ReaderMetadata(bioio_base.reader_metadata.ReaderMetadata):
     """
     Notes
     -----
-    Defines metadata for the reader itself (not the image read), 
+    Defines metadata for the reader itself (not the image read),
     such as supported file extensions.
     """
+
     @staticmethod
     def get_supported_extensions() -> List[str]:
         """
         Return a list of file extensions this plugin supports reading.
         """
-        raise NotImplementedError()
-        # return ["ext", "extn"]
+        return [".tiff"]
```

Comment:
Is this true? There was [this section](https://github.com/AllenCellModeling/aicsimageio/blob/04a92f734dd242a1a761d1064208efac39ed0893/aicsimageio/aics_image.py#L168) in `aicsimageio` that we wanted to revisit for this reader/plug-in. @toloudis or @evamaxfield may be able to way in here on the history of this.

Reply:
I just had a look.  The history is basically that this was an external user's submission for dealing with globbed tiff files where the list of files was a particular data dimension like T. 

I think the only way this plugin could answer the question "what are your supported extensions" is with "tiff".  Clearly we special-cased this reader in the code Sean referenced.

I wonder if we could change this reader's "is_supported_image" to look for the special glob character.   I can't remember what we do when multiple reader plugins report that they support the same file extension.  We have to think about that too.

Reply:
Looks like we sort the reader plugins that support the same extension by install date.  More to consider: If you pass a non-glob path to the glob reader, would it behave normally like a TiffReader?  Could it forward its operations to TiffReader or somehow allow a TiffReader to be created instead of itself?

Reply:
Could we just move responsibility to the user? Since this is from an external user already, we could just have file extensions be ".glob.tiff".

Reply:
Talked in Slack